src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -126,6 +126,9 @@ public function __construct()
             $this->editors['xdebug'] = function ($file, $line) {
                 return str_replace(['%f', '%l'], [$file, $line], ini_get('xdebug.file_link_format'));
             };
+
+            // If xdebug is available, use it as default editor.
+            $this->editor = $this->editors['xdebug'];
```

Comment:
Can we use the method for consistency?

```suggestion
            $this->setEditor('xdebug');
```

Reply:
Of course. That's obviously the better way. Changed. :)